asset/cloud-client/src/main/java/com/example/asset/CreateFeedExample.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.asset;
+
+// [START asset_quickstart_create_feed]
+// Imports the Google Cloud client library
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.asset.v1p2beta1.AssetServiceClient;
+import com.google.cloud.asset.v1p2beta1.CreateFeedRequest;
+import com.google.cloud.asset.v1p2beta1.FeedOutputConfig;
+import com.google.cloud.asset.v1p2beta1.PubsubDestination;
+import com.google.cloud.asset.v1p2beta1.Feed;
+import com.google.cloud.asset.v1p2beta1.Feed;
+import java.util.Arrays;
+
+public class CreateFeedExample {
+
+  // Use the default project Id.
+  private static final String projectId = ServiceOptions.getDefaultProjectId();
+
+  /*
+   * Get a feed with full feed name
+   * @param assetNames used in Feed
+   * @params feed identifier
+   * @params topic name
+   * @param args supplies command-line arguments as an array of String objects.
+   */
+  public static void main(String... args) throws Exception {
```

Comment:
One of the goals for snippets is that they should be [copy-paste-runnable](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#specific-goals). 

This should be a normal function, so that the user can copy-paste this class into an existing project and start iterating it right away. See the [canonical samples](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/dlp/src/main/java/dlp/snippets) for an example of what this should look like.